docs/user/BuildLocally.md

```diff
@@ -46,40 +46,9 @@ make gui_final
 
 ![gcd_final.webp](../images/gcd_final.webp)
 
-## Using Pre-built Binaries
-
-You can download pre-built binaries with self contained dependencies
-included from the Precision Innovations github repository
-[here](https://github.com/Precision-Innovations/OpenROAD/actions/workflows/github-actions-build-deb-package.yml).
-
-Thanks to [Precision Innovations](http://www.precisioninno.com) for hosting and maintaining these binaries.
-
-The following platforms are supported currently:
-- Ubuntu 20.04/22.04
-- Debian 10/11
-
-Use following steps to download:
-
-Step 1: Click on the [Precision Innovations Github releases link](https://github.com/Precision-Innovations/OpenROAD/releases).
-
-Step 2: Download the artifacts for your distribution.
-
-Step 3: Run the install command based on platform use package installer.
-        For example Ubuntu 20.04 use:
-```shell
-sudo apt install ./openroad_2.0_amd64-ubuntu20.04.deb
-```
-
-You can install these binaries within docker as well.
-
-> **Thanks** to Precision Innovations (www.precisioninno.com) for providing
-> and supporting this daily release of pre built installers to the community.
-
 ## Compiling and debugging in Visual Studio Code
 
-Set up environment variables to point to tools that CMake from within
-Visual Studio Code will need, then start Visual Studio Code as usual
-and hit F7, assuming you have CMake plugins installed.
+Set up environment variables using this script, then start Visual Studio Code. Please ensure [CMake plugins](https://code.visualstudio.com/docs/cpp/cmake-linux) are installed.
```

Comment:
> this script

The statement is vague; please be explicit on the script that should be used.
Please also change to:

> Please ensure that the [CMake plugin](https://code.visualstudio.com/docs/cpp/cmake-linux) is installed.